### PR TITLE
Issue 1146

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://github.com/openstax/unified/issues/1146

### CODE REVIEW

**Comment by janKozie1** [comment]
Are you able to create new highlights on this branch? @PiotrKozlowski

**Comment by PiotrKozlowski** [comment]
@janKozie1 I was, but I think I didn't push something and it is now broken... I'll take a look on it when I finish another issue

**Review by janKozie1 — COMMENTED** [review]
The edit card disappears when i try focusing it and there is no annotation. It's probably related to the other issue with creating highlights.

**Review by janKozie1 — COMMENTED** [review]
The discard modal is not showing when I:
 1. Highlight any text
 2. Start writing a note
 3. Highlight any other text

https://drive.google.com/file/d/1ofYPOsYG_uDNNWkZiroShpwtxZquTOQS/view?usp=sharing

File: src/app/content/highlights/components/DisplayNote.tsx [hunk 1]
```diff
@@ -66,9 +66,14 @@ const DisplayNote = React.forwardRef<HTMLElement, DisplayNoteProps>((
     }
   };
 
+  const elements = React.useMemo(
+    () => [element, ...highlight.elements].filter(isElementForOnClickOutside),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [element.current, highlight]);
```
[the remark this answers is not in the document]
Comment (PiotrKozlowski) [reply src/app/content/highlights/components/DisplayNote.tsx:72]: I think passing it as an array of elements is simpler and does not require additional prop

**Comment by PiotrKozlowski** [comment]
@janKozie1 
> The discard modal is not showing when

Thanks, I've pushed a fix for this

File: src/app/content/highlights/components/DisplayNote.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (janKozie1) [reply src/app/content/highlights/components/DisplayNote.tsx:72]: The parameter that this is being passed as to the `onClickOutside` is called an `element`. If it's going to be an element or an array of elements, then the name should probably reflect it too.

Besides that, change of `element.current` won't trigger a rerender so this essentialy hopes that whatever changes `element.current` will cause a rerender as well, which is probably always the case.

**Review by janKozie1 — APPROVED** [review]
It looks good to me

**Pull request merged**